package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "hap-nodejs",
-  "version": "0.9.1",
+  "version": "0.9.2-beta.6",
```

I saw beta 6 was already released? where is the code for that? why is it not pushed to beta branch?
The CI for the beta branch currently works with having a `betaVersion` property defining what the base of the beta version is. In that case it would be `0.9.2`. This way it will always bull in the latest tag number.

```suggestion
  "version": "0.9.1",
  "betaVersion": "0.9.2",
```